File: 프로그래머스/1/12910. 나누어 떨어지는 숫자 배열/나누어 떨어지는 숫자 배열.py
def solution(arr, divisor):
    result = [num for num in arr if (num % divisor == 0)]
    return sorted(result) if (result != []) else [-1]
# result != [] ; 비어 있지 않으면 그냥 true라 result라고 만 써도 됨

File: 프로그래머스/1/12910. 나누어 떨어지는 숫자 배열/나누어 떨어지는 숫자 배열.java
import java.util.*;

class Solution {
    public int[] solution(int[] arr, int divisor) {
        List<Integer> answer = new ArrayList<>();

        Arrays.sort(arr);
        for (int x: arr) {
            if (x % divisor == 0) {
                answer.add(x);
            }
        }
        
        return (!answer.isEmpty()) ? answer.stream().mapToInt(i -> i).toArray() : new int[]{-1};
    }
}